### PR TITLE
Feature/small-changes

### PR DESCRIPTION
**Pull request by kumardennis**
Adding missing texts, fixing some spacing issues, changing pictures.

### CODE REVIEW

File: src/pages/cases.js [hunk 1]
```diff
@@ -53,9 +53,7 @@ const StyledBlogBlock = styled.article`
     padding: 1.5rem;
     margin: 1rem;
     border-top: 2px dotted ${colors.main};
-    h2 {
-        font-size: 1.4rem;
-    }
+    h2 { font-size: 1.6rem; letter-spacing: 0.01em; word-spacing: 0.065em; line-height: 1.2em;}
```
Comment (victorv-sbp) [inline src/pages/cases.js]: Use multiple lines for definition to keep consistency with other classes.

File: src/components/Footer.js [hunk 2]
```diff
@@ -158,6 +160,9 @@ const Footer = () => (
 
                 <div className="footer-somelinks col-4 col-lg-2 col-xl-2 text-right">
                     <SomeIcons className="footer-someicons" color="main" />
+                    <br/>
```
Comment (victorv-sbp) [inline src/components/Footer.js]: Recommended to use styling instead of `<br/>`

**Review by victorv-sbp — CHANGES_REQUESTED** [review]

File: src/components/Footer.js (hunk 2 above)
Comment (kumardennis) [reply src/components/Footer.js]: done

**Comment by edanchenkov** [comment]
@kumardennis Update your PR to point to `develop`

File: src/templates/news-list-template.js [hunk 3]
```diff
@@ -49,49 +49,52 @@ const StyledSelector = styled.button`
 `;
 const StyledBlogBlock = styled.article`
   display: inline-block;
-  width: width: 100%; 
+  width: width: 100%;
```
Comment (edanchenkov) [inline src/templates/news-list-template.js]: That's nice 🙂. I wonder if this is applied and what affect it could have.

**Review by edanchenkov — APPROVED** [review]
There are lot of things we are going to change in the future. Especially in legacy code. But right now it is ok.

**Pull request merged**